minifi/minifi-bootstrap/src/test/java/org/apache/nifi/minifi/bootstrap/status/reporters/StatusLoggerTest.java

```diff
@@ -101,14 +107,16 @@ public void testFailedInitDueToNoQuery(){
     @Test
     public void TestTrace() {
         statusLogger.initialize(getProperties(LogLevel.TRACE), queryableStatusAggregator);
+        statusLogger.setScheduledExecutorService(new RunOnceScheduledExecutorService());
         statusLogger.start();
 
-        verify(logger, Mockito.timeout(300).atLeastOnce()).info(MOCK_STATUS, (Throwable) null);
+        verify(logger, Mockito.timeout(300).atLeastOnce()).trace(MOCK_STATUS, (Throwable) null);
```

Comment:
With the change to ensure that the task is always run, can Mockito.timeout() be removed from all test methods?

Reply:
Yeah would like to do that, it's a moot point now anyway